apps/web/playwright/fixtures/regularBookings.ts

```diff
@@ -635,5 +727,12 @@ export function createBookingPageFixture(page: Page) {
       await page.getByTestId("booking-item").first().getByText("Reject").click();
       await page.getByTestId("rejection-confirm").click();
     },
+    fillEmailAndName: async (eventTypePage: Page) => {
+      const emailLabel = (await localize("en"))("email_address");
+      const nameLabel = (await localize("en"))("your_name");
```

Comment:
We tried to get them by a test ID, but these two components are coming from `packages/features/form-builder/fieldTypes.ts`, a file that does not accept adding a test ID for the element